openmp/libomptarget/DeviceRTL/src/Reduction.cpp

```diff
@@ -194,6 +191,9 @@ int32_t __kmpc_nvptx_teams_reduce_nowait_v2(
     ThreadId = 0;
   }
 
+  uint32_t &IterCnt = state::getKernelLaunchEnvironment().ReductionIterCnt;
+  uint32_t &Cnt = state::getKernelLaunchEnvironment().ReductionCnt;
```

Comment:
These two variables seem not used anywhere else.

Reply:
They are, I replaced the globals with them.